### ISSUE TITLE
--infiles not working

### ISSUE THREAD

**Issue by vforget** [eeertekin/bbcp#15]
I have used bbcp a few times and absolutely love this tool every time I use it.

I am using the -I option to specify a list of input files, but I receive the following error for each file listed:

bbcp: No such file or directory processing <filename>

I have tried to supply relative and absolute paths to no avail. Of note, the same file listing works using rsync with the --files-from option:

rsync --files-from=list.txt . TARGET

Is there a specific spec for the format of fname for --infiles?

**Comment by AGI-chandler** [comment]
This version of bccp is `11.10.05.00.0`, from 2011.  The forker says to check the original page on [standford.edu](https://www.slac.stanford.edu/~abh/bbcp/) for "Complete details", but didn't keep up with newer releases.  Last release of the official bbcp is `15.02.03.00.1` (Feb 2015).  I was having problems the the `-symlinks` option too.

Unfortunately the official version seems to have been abandoned, and their git repository is gone.  Didn't find the source code yet, but there are binaries at https://www.slac.stanford.edu/~abh/bbcp/bin/ still.

The `amd64_linux26` version still works on Linux 4.18 (CentOS), 5.10 (Debian), and 5.18 (Debian) (where I've used it) and is actually version 15.02.03.01.1, but you'll need `libcrypto.so.6`.  Fortunately I still have a CentOS 6 (Linux 2.6) installation going, so just copied the `/usr/lib64/libcrypto.so.6` from there, it works in that bbcp will run, even on the Debian systems.  I think that lib is only used for copying over SSH though?  I haven't used that, mainly use bbcp to copy between servers over NFS.  Either way I attached it to my message here in case it helps. [libcrypto.so.6.zip](https://github.com/eeertekin/bbcp/files/10006725/libcrypto.so.6.zip)

If you're able, would recommend to try one of the binaries.  Let us know if you find the source code somewhere too...

Or if there are alternatives out there that are actively maintained, that'd be good to know too... haven't found anything remotely as fast as bbcp, though.

**Comment by mkll** [comment]
> This version of bccp is `11.10.05.00.0`, from 2011

@AGI-chandler  Actually, this version is later:

```
~$ bbcp --version
14.04.14.00.1
```

**Comment by AGI-chandler** [comment]
> > This version of bccp is `11.10.05.00.0`, from 2011
> 
> @AGI-chandler Actually, this version is later:
> 
> ```
> ~$ bbcp --version
> 14.04.14.00.1
> ```

Yes I could have been mistaken since I was just looking at [MakeSname](https://github.com/eeertekin/bbcp/blob/master/MakeSname), and I've deleted this version and replaced with the newer official binary.

**Comment by mkll** [comment]
I'm sure because I built the binary from this repository. :)

**Comment by AGI-chandler** [comment]
so the forker updated [src/bbcp_Version.C](https://github.com/eeertekin/bbcp/blob/64af83266da5ebb3fdc2f012ac7f5ce0230bc648/src/bbcp_Version.C) but not [MakeSname](https://github.com/eeertekin/bbcp/blob/master/MakeSname) that is shown on the home page.

**Comment by AGI-chandler** [comment]
Actually the original git repository is back up! `git clone https://www.slac.stanford.edu/~abh/bbcp/bbcp.git/` and it's version `17.12.00.00.0`, so see if `--infiles` works with it.
